components/concept/engine/src/main/java/mozilla/components/concept/engine/selection/SelectionActionDelegate.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.selection
+
+/**
+ * Generic delegate for handling the context menu that is shown when text is selected.
+ */
+interface SelectionActionDelegate {
+    /**
```

Comment:
nit: Would prefer a first line explaining what the method does, and then describe what it returns.